machine/machine.go

```diff
@@ -389,6 +392,10 @@ func (m *Machine) RunOnce(i int) (bool, error) {
 		return false, err
 	case kvm.EXITIO:
 		direction, size, port, count, offset := m.runs[i].IO()
+		if port == 0x2eb {
```

Comment:
here's a thought. cf9 is the standard reset port.
"Writing 0xE to 0xCF9:(RESTART) Will power cycle the mother board with everything that comes with it."

I'd advocate that a power cycle in gokvm would mean "exit", since we can't really power cycle? This would be very conformant to standards.

Reply:
I redid it, now it follows reasonable behavior on a write to cf9, a standard port.